src/main/kotlin/org/malachite/estella/services/TaskService.kt

```diff
@@ -25,6 +26,7 @@ import javax.sql.rowset.serial.SerialClob
 @Service
 class TaskService(
     @Autowired private val taskRepository: TaskRepository,
+    @Autowired private val organizationRepository: OrganizationRepository,
     @Autowired private val organizationService: OrganizationService,
```

Comment:
To chyba dosyć mocno jest przeciw DDD, czy jak to się tam zwie. Po to mamy serwisy by nie wołać repozytoriów.

Reply:
Ogólnie to jest mocno nie tyle przeciw DDD, tylko ogólnie tak się nie robi. Mamy serwisy, które robią to lepiej, niż repozytorium - jak jakiejś metody Ci brakuje, to doklep w serwisie XD